packages/catalog-search/src/SearchSuggestionItem.jsx

```diff
@@ -3,43 +3,56 @@ import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
 
 const SearchSuggestionItem = ({
-  title, url, highlightedTitle, authoringOrganization, programType,
-}) => (
-  <Link to={url} key={title} className="suggestion-item">
-    <div>
-      {/* eslint-disable-next-line react/no-danger */}
-      <div dangerouslySetInnerHTML={{ __html: highlightedTitle }} />
-      {
+  url, suggestionItemHandler, hit, disableSuggestionRedirect,
+}) => {
+  const authoringOrganization = hit.key && hit.key.split('+')[0];
+  // If the disable redirect bool is provided, prevent the redirect from happening and instead call the provided submit
+  // handler
+  const handleLinkDisable = (e) => {
+    if (disableSuggestionRedirect) {
```

Comment:
I think we may be able to remove the disableSuggestionRedirect flag and instead use the validity of the suggestionItemHandler to replace this check? What do you think?

Reply:
talked offline and decided to resolve this in a fast follow. Ticket TBD